playwright/src/main/java/com/microsoft/playwright/Browser.java

```diff
@@ -25,7 +25,7 @@
  *
  * <p> A Browser is created via [{@code method: BrowserType.launch}]. An example of using a {@code Browser} to create a [Page]:
  */
-public interface Browser {
+public interface Browser extends AutoCloseable {
```

Comment:
These classes are generated, can you update [api-generator](https://github.com/microsoft/playwright-java/blob/master/tools/api-generator/src/main/java/com/microsoft/playwright/tools/ApiGenerator.java) with the new base interface? You can have a look at [`generate_api.sh`](https://github.com/microsoft/playwright-java/blob/master/scripts/generate_api.sh) to see how the generator is launched.  If the setup seems too tricky it's fine to commit it with the changes in the generated interfaces too and I can update the generator with the next roll of the playwright cli (I expect more changes there anyway).